manifests/service/cors/configure.pp

```diff
@@ -81,9 +78,6 @@
         owner   => $owner,
         group   => $group,
         content => template('lightblue/properties/lightblue-cors.json.erb'),
-        before  => $before,
-        notify  => $notify,
-        require => $require,
```

Comment:
Oh, nice! I'll go ahead and make that change today unless you find out it doesn't work. (I'll poke around that documentation too)